sidebars.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       "getting-started/authentication",
       "getting-started/connect-server-to-github",
       "getting-started/helm-chart",
-      "getting-started/workspace",
```

Comment:
did you want to override the workspace page? or did you want to add a new page?

Reply:
@yuval-hazaz Workspace should have been part of another branch.  Not sure what it is doing here.

Reply:
> @yuval-hazaz Workspace should have been part of another branch. Not sure what it is doing here.

This was included in PR  #159 which was already merged. It should be neither overidden nor should a new page be added. What should I do to resolve this?